docs/sources/writing-guide/front-matter/index.md

```diff
@@ -44,14 +44,12 @@ The following headings describe what each element does and provides guidelines f
 
 ### aliases
 
-Provides an HTML redirect from the pages in the list to the current page.
-For more information, refer to [Hugo aliases](#hugo-aliases).
+Used to create redirects from the previous URL to the new URL when a page changes or moves. As a best practice, when you rename or move files, you should create an alias with the previous file path and name to create an HTML redirect from the old URL to the new URL. For more information, refer to [Hugo aliases](#hugo-aliases).
 
-### description
+### (required) description
 
-**Required.**
-
-On social media, such as Twitter, displays as a clue to users about what the page includes.
+Used to provide the short description of the topic to search engines, including the search feature in the Grafana documentation site.
```

Comment:
I immediately thought you were talking about the feature called "Search" in Grafana. Mixing language here confuses the issue I think. 
```suggestion
Used to provide the short description of the topic to search engines, including the search engine used in the Grafana documentation site.
```